shadows/framework/src/main/java/org/robolectric/shadows/ShadowCryptoObject.java

```diff
@@ -0,0 +1,17 @@
+package org.robolectric.shadows;
+
+import static android.os.Build.VERSION_CODES.P;
+
+import android.hardware.biometrics.CryptoObject;
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+@SuppressWarnings("UnusedDeclaration")
+@Implements(value = CryptoObject.class, minSdk = P)
+public class ShadowCryptoObject {
```

Comment:
@alwa Do you decide to use shadow to fix errors? I think @hoisie can help to bump pre-instrumentation jar version if you use the previous patch. But before hoisie help to bump jar version, I suppose you can write a test to test it as Robolectric CI can use non-preinstrumented jar to test your patch with your tests.

Reply:
Yes. I have a test in this branch that failed before adding the shadow (locally)

Reply:
I have reverted the previous patch